linux_os/guide/system/permissions/restrictions/daemon_umask/umask_for_daemons/rule.yml

```diff
@@ -8,8 +8,7 @@ description: |-
     The file <tt>/etc/init.d/functions</tt> includes initialization
     parameters for most or all daemons started at boot time.  The default umask of
     022 prevents creation of group- or world-writable files.  To set the default
```

Comment:
If `var_umask_for_daemons` is tailored to `027`, this snippet would look out of place `The default umask of 022 prevents creation of group- or world-writable files. `

Reply:
@yuumasato I think it is important to keep it as many people get confused by `umask`, so I refactored the script to hopefully make more sense now as well as highlight the affects of changing the umask.